dir.props

```diff
@@ -85,6 +85,8 @@
 
   <!-- Test runtime -->
   <PropertyGroup>
+    <TestRuntimeProjectJson Condition="'$(TestRuntimeProjectJson)' == '' And '$(TargetGroup)' == 'net46'">$(SourceDir)Common/test-runtime/desktop/project.json</TestRuntimeProjectJson>
```

Comment:
Can you please follow the naming convention and name the directory net46 instead of desktop.